sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
 ## 1.2.0-beta.2 (Unreleased)
-- Reverted change in 1.2.0-beta.1 which moved @rollup/plugin-json from devDependencies to dependencies
+- Reverted change in 1.2.0-beta.1 which moved @rollup/plugin-json from devDependencies to dependencies.
+- `DefaultAzureCredential` now by default shows the Device Code message on the console. This can still be overwritten with a custom behavior by specifying a function as the third parameter, `userPromptCallback`.
```

Comment:
I would swap the order of changes just to bring more attention to the feature change first and bug fix later

Reply:
I believe that if I swap it before addressing https://github.com/Azure/azure-sdk-for-js/pull/11475/files#r494596700 it will make things weird on GitHub, so I'll swap this only after addressing that other concern.

Reply:
Done!